src/QirRuntime/test/QIR-static/qsharp/qir-gen.csproj

```diff
@@ -3,7 +3,9 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp3.1</TargetFramework>
+    <CSharpGeneration>false</CSharpGeneration>
     <QirGeneration>True</QirGeneration>
+    <ExecutionTarget>honeywell.qpu</ExecutionTarget>

```

Comment:
Can you please provide some more details about this line?

Reply:
This is the target that triggers conditional rewrites in the compiler. As discussed in https://github.com/microsoft/qsharp-runtime/pull/527#discussion_r581454320 I'm removing it.